core/common/modules/connect/module.php

```diff
@@ -198,4 +198,16 @@ public function get_subscription_plans( $context ) {
 		];
 	}
 
+	protected function get_init_settings() {
```

Comment:
@Nevoss it's something you changed in another PR?

Reply:
@matipojo no, @nisan250 needs to know if the user is connected or no to show the "connect" button in the top bar, although it can be useful for the kit library.